samples/net/sockets/sntp_client/src/main.c

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2017 Linaro Limited
+ * Copyright (c) 2019 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(net_sntp_client_sample, LOG_LEVEL_DBG);
+
+#include <zephyr.h>
+#include <misc/printk.h>
+
+#include <net/socket.h>
+#include <net/net_ip.h>
+#include <net/sntp.h>
```

Comment:
Small improvement: Only `#include <net/sntp.h>` (and the logging system of course) is required for the sample to work.